Src/Screens/ProfileScreen/Screens/FeedbackScreen/Feedback.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Feedback.swift
+//  Hedvig
+//
+//  Created by Gustaf Gunér on 2019-02-14.
+//  Copyright © 2019 Hedvig AB. All rights reserved.
+//
+
+import Flow
+import Form
+import Presentation
+import UIKit
+
+struct Feedback {
+    let presentingViewController: UIViewController
```

Comment:
As this is a presentable, presentingViewController will be itself, so remove this and on line 39 just use viewController.

Reply:
Yeah, that makes sense 👍